queries/flights.py

```diff
@@ -0,0 +1,502 @@
+
+            # Make sure flight isn't in the past
+            if flight_time and flight_time >= now:
+                flight["flight_time"] = flight_time
+                flight["api_airport"] = api_airport
+
+                flight_info.append(flight)
+                n -= 1
+
+    return flight_info
+
+
+_BREAK_LENGTH = 0.5  # Seconds
+_BREAK_SSML = '<break time="{0}s"/>'.format(_BREAK_LENGTH)
+
+
+def _format_flight_answer(flight_info: List[Dict[str, Any]]) -> Dict[str, str]:
+    """
+    Takes in a list of flights and returns a dict
+    containing a formatted answer and text for a voice line.
+
+    Each flight should contain the attributes:
+        'No':           Flight number
+        'DisplayName':  Name of airport/city
+        'api_airport':  Name of Icelandic airport/city
+        'flight_time':  Time of departure/arrival
+        'Departure':    True if departing from api_airport, else False
+        'Status':       Info on flight status (e.g. whether it's cancelled)
+    """
+    airport: str
+    api_airport: str
+    flight_dt: datetime
+    answers: List[str] = []
+    voice_lines: List[str] = []
+
+    for flight in flight_info:
+        airport = icelandic_city_name(capitalize_placename(flight["DisplayName"]))
+        api_airport = icelandic_city_name(capitalize_placename(flight["api_airport"]))
+
+        flight_dt = flight["flight_time"]
+        flight_date_str = flight_dt.strftime("%-d. %B")
+        flight_time_str = flight_dt.strftime("%H:%M")
+
+        if flight["Departure"]:
+            airport = NounPhrase(airport).genitive or airport
+            api_airport = NounPhrase(api_airport).dative or api_airport
+
+            # Catch cancelled flights
+            if flight["Status"] and "aflýst" in flight["Status"].lower():
+                line = (
+                    f"Flugi {flight['No']} frá {api_airport} til {airport} er aflýst."
+                )
+            else:
+                line = (
+                    f"Flug {flight['No']} til {airport} "
+                    f"flýgur frá {api_airport} {flight_date_str} "
+                    f"klukkan {flight_time_str} að staðartíma."
+                )
+        else:
+            airport = NounPhrase(airport).dative or airport
+            prep = iceprep_for_placename(api_airport)
+            api_airport = NounPhrase(api_airport).dative or api_airport
+
+            if flight["Status"] and "aflýst" in flight["Status"].lower():
+                line = (
+                    f"Flugi {flight['No']} frá {airport} til {api_airport} er aflýst."
+                )
+            else:
+                line = (
+                    f"Flug {flight['No']} frá {airport} "
+                    f"lendir {prep} {api_airport} {flight_date_str} "
+                    f"klukkan {flight_time_str} að staðartíma."
+                )
+
+        voice_line = re.sub(r" \d+\. ", " " + _DAY_INDEX_ACC[flight_dt.day] + " ", line)
```

Comment:
This normalization is great - and would be good to have in a common library function, possibly in `__init__.py` in the queries directory.

Reply:
It would be very handy to have a function that turns integers into written Icelandic ordinals (for voice lines and such), is there any such functionality somewhere? (Otherwise I could attempt to create one in another pull request)

Reply:
No, I've been meaning to implement something along those lines for a while now, but feel free to take a stab at it!